pkgs/by-name/si/sigil/package.nix

```diff
@@ -30,37 +26,47 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [
     cmake
     pkg-config
-    makeWrapper
+    qt6.wrapQtAppsHook
   ];
 
   buildInputs = [
     boost
     xercesc
-    qtbase
-    qttools
-    qtwebengine
-    qtsvg
+    qt6.qtbase
+    qt6.qttools
+    qt6.qtwebengine
+    qt6.qtsvg
     python3Packages.lxml
   ];
 
   prePatch = ''
     sed -i '/^QTLIB_DIR=/ d' src/Resource_Files/bash/sigil-sh_install
   '';
 
-  dontWrapQtApps = true;
+  installPhase = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    mv bin/Sigil.app $out/Applications
+    # https://github.com/NixOS/nixpkgs/issues/186653
+    chmod -x $out/Applications/Sigil.app/Contents/lib/*.dylib \
+      $out/Applications/Sigil.app/Contents/polyfills/*.js \
+      $out/Applications/Sigil.app/Contents/python3lib/*.py \
+      $out/Applications/Sigil.app/Contents/hunspell_dictionaries/*.{aff,dic}
+
+    runHook postInstall
+  '';
 
   preFixup = ''
-    wrapProgram "$out/bin/sigil" \
-       --prefix PYTHONPATH : $PYTHONPATH \
-       ''${qtWrapperArgs[@]}
+    qtWrapperArgs+=(--prefix PYTHONPATH : "$PYTHONPATH")
   '';
 
   meta = {
     description = "Free, open source, multi-platform ebook (ePub) editor";
     homepage = "https://github.com/Sigil-Ebook/Sigil/";
     license = lib.licenses.gpl3;
```

Comment:
```suggestion
    license = lib.licenses.gpl3Plus;
```

Reply:
Could you give some references?

Reply:
https://github.com/Sigil-Ebook/Sigil/blob/6afd1845be52966a75793bf762a11cc1ebe52a49/src/MainUI/ClipsWindow.cpp#L10-L11